deployment-templates/compute-engine/forseti-instance.py

```diff
@@ -139,95 +139,92 @@ def GenerateConfig(context):
 fi
 
 # Check whether Cloud SQL proxy is installed.
-CLOUD_SQL_PROXY=$(ls $USER_HOME/cloud_sql_proxy)
+CLOUD_SQL_PROXY=$(which cloud_sql_proxy)
 if [ -z "$CLOUD_SQL_PROXY" ]; then
         cd $USER_HOME
-        wget https://dl.google.com/cloudsql/cloud_sql_proxy.{}
-        mv cloud_sql_proxy.{} cloud_sql_proxy
-        chmod +x cloud_sql_proxy
+        wget https://dl.google.com/cloudsql/cloud_sql_proxy.{cloudsql_arch}
+        sudo mv cloud_sql_proxy.{cloudsql_arch} /usr/local/bin/cloud_sql_proxy
+        chmod +x /usr/local/bin/cloud_sql_proxy
 fi
 
-$USER_HOME/cloud_sql_proxy -instances={}=tcp:{} &
-
 # Install Forseti Security.
 cd $USER_HOME
 rm -rf *forseti*
 pip install --upgrade pip
 pip install --upgrade setuptools
 pip install grpcio==1.4.0 grpcio-tools==1.4.0 google-apputils
```

Comment:
I don't think we need to lock the grpcio version down any more. That seems to have been a temporary bug.